src/main/java/ch/njol/skript/effects/EffFreezeServer.java

```diff
@@ -0,0 +1,70 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.bukkitutil.ServerUtils;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+@Name("Freeze/Unfreeze Server")
+@Description("Freezes or unfreezes the server.")
+@Examples({
+	"freeze server",
+	"unfreeze server"
+})
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20.4+")
+public class EffFreezeServer extends Effect {
+
+
+	static {
+		if (Skript.methodExists(Bukkit.class, "getServerTickManager"))
+			Skript.registerEffect(EffFreezeServer.class,
+				"freeze [the] server",
+				"unfreeze [the] server");
```

Comment:
```suggestion
			Skript.registerEffect(EffFreezeServer.class, "[:un]freeze [the] server");
```
Pattern could be simplified to this (update the logic in init accordingly)